stable/rabbitmq/templates/deployment.yaml

```diff
@@ -42,6 +42,8 @@ spec:
           value: {{ default "/" .Values.rabbitmqVhost | quote }}
         - name: RABBITMQ_MANAGER_PORT_NUMBER
           value: {{ default "15672" .Values.rabbitmqManagerPort | quote }}
+        - name: RABBITMQ_DISK_FREE_LIMIT
+          value: {{ default '"8GiB"' .Values.rabbitmqDiskFreeLimit | quote }}
```

Comment:
The Yaml parser doesn't like this.

`Error: parse error in "rabbitmq/templates/deployment.yaml": template: rabbitmq/templates/deployment.yaml:46: malformed character constant: '"8GiB"'`

Reply:
fixed it